documentation/src/test/java/org/hibernate/search/documentation/search/predicate/LucenePredicateDslIT.java

```diff
@@ -44,10 +49,56 @@ class LucenePredicateDslIT {
 
 	@BeforeEach
 	void setup() {
-		entityManagerFactory = setupHelper.start().setup( Book.class, Author.class, EmbeddableGeoPoint.class );
+		entityManagerFactory = setupHelper.start()
+				.withProperty(
+						HibernateOrmMapperSettings.MAPPING_CONFIGURER,
+						(HibernateOrmSearchMappingConfigurer) context -> context.programmaticMapping()
+								.type( Book.class )
+								.property( "coverImage" )
+								.vectorField( 128 )
+				)
```

Comment:
Adding an annotation to the entity will lead to Elasticsearch suite failing all the tests at init stage, since it doesn't support the annotation so far... so the idea is to keep the same entities and map the vector only in Lucene-specific IT.

Reply:
Ok, makes sense... Could you move this all to PredicateDslIT and make the mapping + test conditional, so it's clearer?